Sources/IdentityState.swift

```diff
@@ -175,6 +206,46 @@ class IdentityState {
         return true
     }
 
+    /// Determines if we should update the advertising identifier with `newAdID` and if the advertising tracking consent has changed.
```

Comment:
Minor rewording: Determines if the advertising identifier should be updated with `newAdID` ...